diff --git a/dist/bin.js b/dist/bin.js
index a8a878106bac35426cafd1319235459f42f8b7df..d047f466ab7bc4727b7a99f68fbdbc239dcab621 100644
--- a/dist/bin.js
+++ b/dist/bin.js
@@ -996,7 +996,7 @@ function getCustomType(field) {
   const [module2, type] = splits;
   return {
     imports: namedImport([type], module2),
-    code: `.$type<${type}>()`
+    code: `.$type<${type}${field.isList ? '[]' : ''}>()`
   };
 }
 function getCustomDefault(field) {
